### PR TITLE
(SDK-329) implement running arbitrary commands in PDK's environment

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,13 +270,36 @@ Multiple `--format` options can be specified as long as they all have distinct o
 
 Specifies options to pass through to the actual test-runner. In the default template (and most commonly across modules), this is [rspec](https://relishapp.com/rspec/rspec-core/docs/command-line).
 
+#### `pdk bundle` command
```
Comment (james-stocks) [inline README.md]: I think this is indented one hash too many

File: lib/pdk/cli/bundle.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (james-stocks) [inline lib/pdk/cli/bundle.rb]: I think this blank line shouldn't be here; I'm surprised rubocop doesn't complain about it

File: README.md [hunk 3]
```diff
@@ -270,13 +270,36 @@ Multiple `--format` options can be specified as long as they all have distinct o
...
+
+This experimental command allows you to execute arbitrary commands in a bundler context within the module you're currently working on. Arguments to this command are passed straight through to bundler.
```
Comment (james-stocks) [inline README.md]: The command text says this is for 'advanced users' - we might want at least that level of warning here; if not a stronger warning that the user should know what they're doing (e.g. they might install a gem to system with this command, causing problems on their machine)?

File: lib/pdk/cli/bundle.rb (hunk 2 above)
Comment (DavidS) [inline lib/pdk/cli/bundle.rb]: this is not the only file to have this.

File: README.md [hunk 4]
```diff
@@ -270,13 +270,36 @@ Multiple `--format` options can be specified as long as they all have distinct o
 
 Specifies options to pass through to the actual test-runner. In the default template (and most commonly across modules), this is [rspec](https://relishapp.com/rspec/rspec-core/docs/command-line).
 
+### `pdk bundle` command
+
+This experimental command allows you to execute arbitrary commands in a bundler context within the module you're currently working on. Arguments to this command are passed straight through to bundler.
+
+Note that for most advanced uses it is required to use the `--` to separate bundler options from pdk options. Compare the following two commands:
```
Comment (james-stocks) [inline README.md]: The command text says this is for 'advanced users' - we might want at least that level of warning here; if not a stronger warning that the user should know what they're doing (e.g. they might install a gem to system with this command, causing problems on their machine)?

File: README.md (hunk 3 above)
Comment (DavidS) [inline README.md]: ha, I copied it from the one heading that had it wrong. fixed both now.

File: README.md (hunk 4 above)
Comment (DavidS) [inline README.md]: Good call. I'll make the language a bit stronger.

**Comment by DavidS** [comment]
Updated, addressing all recommendations.

**Pull request merged**